spec/requests/users_controller_spec.rb

```diff
@@ -423,7 +423,7 @@
       context "when a search parameter is passed" do
         let!(:other_user_2) { FactoryBot.create(:user, organisation: user.organisation, name: "joe", email: "other@example.com") }
         let!(:other_user_3) { FactoryBot.create(:user, name: "User 5", organisation: user.organisation, email: "joe@example.com") }
-        let!(:other_org_user) { FactoryBot.create(:user, name: "User 4", email: "joe@other_example.com") }
```

Comment:
Why removing the `_` here?

Reply:
Because it's not a valid character for a domain, right...

Reply:
Yup, previously validations were't working properly, hence this was accepted.